agent/handlers/v4/response.go

```diff
@@ -187,3 +189,40 @@ func newNetworkInterfaceProperties(task *apitask.Task) (NetworkInterfaceProperti
 		SubnetGatewayIPV4Address: eni.SubnetGatewayIPV4Address,
 	}, nil
 }
+
+// NewPulledContainerResponse creates a new v4 container response for a pulled container.
+// It augments v4 container response with an additional empty network interface field.
+func NewPulledContainerResponse(
+	dockerContainer *apicontainer.DockerContainer,
+) ContainerResponse {
+	resp := toV4PulledContainerResponse(dockerContainer)
+	return ContainerResponse{
+		ContainerResponse: &resp,
+		Networks:          []Network{},
+	}
+}
+
+// toV4PulledContainerResponse returns v2 container response to v4 for a pulled container.
+func toV4PulledContainerResponse(
```

Comment:
hmm, for the sake of DRY, I think I'd prefer to export `v2.newContainerResponse` instead of duplicating it